feature(example): Add Act_Web_API example (#267)
fivetran/fivetran_connector_sdk: README.md

<details class="details-heading" open="open">
<summary><h3>Source examples</h3></summary>

> - [act_web_api] (https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/act_web_api) - This is an example of how we can connect to an Act! Premium for Web API instance and pull 6 endpoints (contacts, companies, opportunities, activities, activity_types, products).

Reviewer: ```suggestion
- [act_web_api] (https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/act_web_api) - This is an example of how to connect to an Act! Premium for Web API instance and sync data from 6 endpoints, contacts, companies, opportunities, activities, activity_types, and products.
```
